src/Compiler/AbstractIL/il.fs

```diff
@@ -4358,7 +4358,7 @@ let buildILCode (_methName: string) lab2pc instrs tryspecs localspecs : ILCode =
 // Detecting Delegates
 // --------------------------------------------------------------------
 
-let mkILDelegateMethods access (ilg: ILGlobals) (iltyp_AsyncCallback, iltyp_IAsyncResult) (parms, rtv: ILReturn) =
+let mkILDelegateMethods access (ilg: ILGlobals) (iltyp_AsyncCallback, iltyp_IAsyncResult) (params_, rtv: ILReturn) =
```

Comment:
can also do "parameters"

Reply:
Please, don't rename any of the variables in compiler. Most of them are very well established, and easier to search/look for.

Reply:
"parms" is not very common but yeah it appears elsewhere. "valu" is another case, definitely keep.